frontend-react/src/utils/ScreenWidth.ts

```diff
@@ -0,0 +1,11 @@
+export default function isScreenWidth(size: "mobile" | "tablet" | "desktop") {
+    const currentScreenWidth = window.innerWidth;
+
+    if (size === "mobile") {
+        return currentScreenWidth <= 480;
```

Comment:
Hmmm this is a minor point, but I'm a little wary of hardcoded values like this in our JavaScript when they're actually defined [in the USWDS CSS](https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/styles/tokens/units/spacing.scss).  We could write a utility function to determine which breakpoint we're currently on: add an invisible element, add a breakpoint-specific modifier class, check if it's applied.  Otherwise, I think we should make these values enums so that 1) they're in one place for easier modification and 2) we can reference them to pass into this function -- and possibly others in the future.  

What do you think?

Reply:
It seems like we define our own set of breakpoints in `_uswds_settings.scss` with:

https://github.com/CDCgov/prime-reportstream/blob/dc43aea6730b930634c5732f018fa464b1d376bf/frontend-react/src/styles/_uswds_settings.scss#L48-L51

So I could see defining those breakpoints in one place like you suggested!